### DISCUSSION TITLE
AdGuard LXC update fails - wget unrecognised option

### DISCUSSION THREAD

**Discussion by ItsMrChops** [tteck/Proxmox#1504] [Help Needed] [answered]
Hi @tteck, AdGuard has been running great in Proxmox (installed from your script) and today it says an update is available.
I'm a bit of a newbie with Proxmox & containers (and Linux), so I checked the instructions and saw that to upgrade I should run the command `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/adguard.sh)"` in the LXC Console.

The response was the following error:

```shell
alpine-adguard:~# bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/adguard.sh)"
wget: unrecognized option: L
BusyBox v1.36.0 (2023-04-11 16:01:21 UTC) multi-call binary.

Usage: wget [-cqS] [--spider] [-O FILE] [-o LOGFILE] [--header STR]
        [--post-data STR | --post-file FILE] [-Y on/off]
        [-P DIR] [-U AGENT] [-T SEC] URL...

Retrieve files via HTTP or FTP

        --spider        Only check URL existence: $? is 0 if exists
        --header STR    Add STR (of form 'header: value') to headers
        --post-data STR Send STR using POST method
        --post-file FILE        Send FILE using POST method
        -c              Continue retrieval of aborted transfer
        -q              Quiet
        -P DIR          Save to DIR (default .)
        -S              Show server response
        -T SEC          Network read timeout is SEC seconds
        -O FILE         Save to FILE ('-' for stdout)
        -o LOGFILE      Log messages to FILE
        -U STR          Use STR for User-Agent header
        -Y on/off       Use proxy
alpine-adguard:~#
``` 
I'm running the command in the container's **>_ Console** window. My Proxmox is all up to date.
Seems sort of similar to #711 but this command worked for them. Any help with what I'm doing wrong would be appreciated.
Thanks!

**Comment by tteck** [comment, marked as answer]
1) Adguard should be updated from within its GUI.
2) Unfortunately, **alpine-adguard** is not supported in this repository since May 17th. https://github.com/tteck/Proxmox/commit/9ca7ef55ddfa076b9ff57515b8624dd299cc86ea
3) There was never a manual update path.

I would suggest changing to the regular installation.

**Reply by ItsMrChops** [reply]
Thank you for the reply.
